.github/workflows/sdk_protos_map.csv

```diff
@@ -38,18 +38,18 @@ board,GetPWM,Yes,get_pwm,PWM,pwm
 board,SetPWM,Yes,set_pwm,SetPWM,setPwm
 board,PWMFrequency,Yes,get_pwm_frequency,PWMFreq,pwmFrequency
 board,SetPWMFrequency,Yes,set_pwm_frequency,SetPWMFreq,setPwmFrequency
-board,AnalogByName,No,analog_by_name,AnalogByName,analogReaderValue
-## HACK: Omitting PySDK: write_analog, currently borked: https://python.viam.dev/autoapi/viam/components/board/client/index.html#viam.components.board.client.BoardClient.write_analog
-board,Write,Yes,,Write,writeAnalog
-board,GetDigitalInterruptValue,No,digital_interrupt_by_name,DigitalInterruptByName,,
+board,GetDigitalInterruptValue,No,value,Value,digitalInterruptValue
+board,ReadAnalogReader,Yes,read,Read,analogReaderValue
```

Comment:
This is supported by micro-rdk? You're sure?

Reply:
Was just going by what the old “read” entry said, but changing the proto name to the actual proto name